packages/components/package.json

```diff
@@ -49,7 +49,7 @@
     "react": ">= 16.8.0"
   },
   "dependencies": {
-    "@chanzuckerberg/eds-tokens": "^0.1.2",
+    "@chanzuckerberg/eds-tokens": "workspace:*",
```

Comment:
Instead of specifying a version, we tell Yarn to only match the version in the monorepo.

Doing this doesn't directly solve https://github.com/lerna/lerna/issues/1171, but works around it because we no longer need the yarn.lock file to be updated when publishing.

Also note that https://github.com/lerna/lerna/issues/2564 means this change will cause any `lerna run` calls to fail. That should be fine, because we can run anything through `yarn workspaces foreach`.

I will test that publishing still works.